app/templates/index.js

```diff
@@ -5,6 +5,24 @@ const BrowserWindow = require('browser-window');
 // report crashes to the Electron project
 require('crash-reporter').start();
 
+function createMainWindow () {
+	let browserWin = new BrowserWindow({
+		width: 600,
+		height: 400,
+		resizable: false
+	});
+
+	browserWin.loadUrl(`file://${__dirname}/index.html`);
+
```

Comment:
maybe register the `mainWindow.on('closed', onClosed);` here so you don't have to do it twice?